load-test/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "chefs-load-test",
+  "version": "1.0.0",
```

Comment:
Let's have this be version 0.1.0 to keep it aligned with the impending release.

Reply:
Not going to coordinate this package with other package versions... no one will keep it up to date.